src/docs/product/issues/performance-issues/n-one-queries.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "N+1 Queries"
+sidebar_order: 10
+description: "Learn more about N+1 queries and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+## N+1 Queries
+
+"N+1 Queries" is a performance problem where the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection has some amount of performance overhead, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when using an Object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
+
+## Span Evidence
+
+The evidence for a N+1 queries has four main aspects:
+
+- the transaction name
+- the parent span. This can be a view, a serializer, or another span that logically groups the queries
+- the source span. This is the query that caused the N+1 queries
+- the repeating span. This is the "N" of "N+1 queries". This is the looped query that should have been part of the source
+
+![N+1 Query span evidence](span-evidence.png)
+
+## Example
+
+Consider a book review website. It has two ORM models: `Book` and `Author`, each with a corresponding database table. The website shows a list of ten oldest books and their respective authors. The code might look like this:
+
+```python
+from django.http import HttpResponse
+
+def books(request):
+    books = Book.objects.all()[:10]
+    book_list = [book.title + " by " + book.author.name for book in books]
+    return HttpResponse((", ").join(book_list))
+```
+
+This code has a subtle performance problem. Each call to `book.author.name` makes a query to fetch the book's author! In total, this code makes 11 queries: one query to fetch the list of books, and 10 more queries to fetch the author of each book. This results in a characteristic query span waterfall:
+
+![N+1 queries in an example application](n-plus-one-database-queries-in-booker.png)
+
+In order to fix this issue, you could use the `selected_related` method in Django, like so:
```

Comment:
I read about using prefetch related as well to help solve N+1
https://stackoverflow.com/questions/31237042/whats-the-difference-between-select-related-and-prefetch-related-in-django-orm

tbh, i'm not super familiar with Django, so I'm not proposing any change to docs, but perhaps something to consider.

Reply:
Noted, thanks!